autogpt_platform/frontend/src/components/agptui/StoreCard.tsx

```diff
@@ -72,37 +72,46 @@ export const StoreCard: React.FC<StoreCardProps> = ({
         )}
       </div>
 
-      {/* Content Section */}
-      <div className="w-full px-2 py-4">
-        {/* Title and Creator */}
-        <h3 className="mb-0.5 font-poppins text-2xl font-semibold text-[#272727] dark:text-neutral-100">
-          {agentName}
-        </h3>
-        {!hideAvatar && creatorName && (
-          <p className="mb-2.5 font-sans text-xl font-normal text-neutral-600 dark:text-neutral-400">
-            by {creatorName}
+      <div className="mt-[12px] flex w-full flex-1 flex-col px-4">
```

Comment:
```suggestion
      <div className="mt-3 flex w-full flex-1 flex-col px-4">
```
Better to not use pixels

Reply:
Done..!!